api/all.yaml

```diff
@@ -2278,6 +2375,138 @@ components:
       required:
         - principal
         - privileges
+    Commit:
```

Comment:
Please add what properties are required

https://stackoverflow.com/questions/40113049/how-to-specify-if-a-field-is-optional-or-required-in-openapi-swagger

Reply:
This is already added after properties